knimin/lib/data_access.py

```diff
@@ -3197,6 +3197,59 @@ def get_property_options(self, prop):
                                prop + '_id'))
             return [dict(x) for x in TRN.execute_fetchindex()]
 
+    def get_or_create_property_option_id(self, prop, option_name):
```

Comment:
This function does two things, which can be separated into two functions. Will it be possible that the user wants to create a new property (extraction kit lot?) not with a plate, but somehow _de novo_?